app/assets/stylesheets/users.css.scss

```diff
@@ -1,3 +1,26 @@
 // Place all the styles related to the Users controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.btn-email-pref {
+  color: #fff;
+  background: #444;
+  border-color: rgba(0, 0, 0, 0.2);
+  border-radius: 15px;
+  font-size: 25px;
+  line-height: 1.5em;
+  padding-left: 51px;
+  padding-right: 51px;
+  margin-top: 5px;
+}
+
+@media(max-width: 480px){
+  .btn-email-pref{
+    margin-left: 0;
+  }
+}
+
+.btn-email-pref:hover {
+  background: #303030;
+  color: #fff;
+}
```

Comment:
you have a set up your text editor to include a new blank line at the end the file when you saving.